NGitLab/Models/References.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Text.Json.Serialization;
+
+namespace NGitLab.Models;
+
+public class References
+{
+    [JsonPropertyName("short")]
+    public string Short;
```

Comment:
Please define these public fields as properties instead.

Reply:
Done, but is there a reason why other classes like User, Issue, Author, Assignee and Milestone are not using properties? (I used fields to be consistent)

Reply:
Older types use fields but when introducing new types we try to enforce use of properties.